R/input-action.R

```diff
@@ -37,6 +37,10 @@
 #' }
 #'
 #' @seealso [observeEvent()] and [eventReactive()]
+#'
+#' @section Server value:
+#' An integer, initialized as 0 and incremented with each press.
```

Comment:
It might be worth noting that this `0` is slightly different than any other `0` value, in that it is considered falsy (see `?req`). Basically, `req(input$btn)` on an unpressed button should throw, while `req(input$num)` where `input$num==0` should not.